x-pack/legacy/plugins/ml/public/components/navigation_menu/main_tabs.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useState } from 'react';
+import { EuiTabs, EuiTab, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import chrome from 'ui/chrome';
+import { TabId } from './navigation_menu';
+
+export interface Tab {
+  id: TabId;
+  name: any;
+  disabled: boolean;
+}
+
+interface Props {
+  disableLinks: boolean;
+  tabId: TabId;
+}
+
+function getTabs(disableLinks: boolean): Tab[] {
+  return [
+    // {
+    //   id: 'overview',
+    //   name: i18n.translate('xpack.ml.navMenu.overviewTabLinkText', {
+    //     defaultMessage: 'Overview',
+    //   }),
+    //   disabled: disableLinks,
+    // },
+    {
+      id: 'anomaly_detection',
+      name: i18n.translate('xpack.ml.navMenu.anomalyDetectionTabLinkText', {
+        defaultMessage: 'Anomaly Detection',
+      }),
+      disabled: disableLinks,
+    },
+    {
+      id: 'data_frames',
+      name: i18n.translate('xpack.ml.navMenu.dataFrameTabLinkText', {
+        defaultMessage: 'Transforms',
+      }),
+      disabled: false,
+    },
+    {
+      id: 'data_frame_analytics',
+      name: i18n.translate('xpack.ml.navMenu.dataFrameAnalyticsTabLinkText', {
+        defaultMessage: 'Data Frame Analytics',
```

Comment:
Maybe we could also add a breadcrumbs file in for the Analytics page, to add in breadcrumbs for `Machine Learning /
Analytics / Job Management` and `Machine Learning / Analytics / Exploration`? Currently once you get to the Exploration page, the only way back to the Jobs List is to click the Data Frame Analytics menu.

![image](https://user-images.githubusercontent.com/7405507/64696372-25733180-d496-11e9-96c4-e043f5e4d000.png)

Reply:
Added `Job Management`/`Exploration` extensions to these breadcrumbs in 047c7a71b68138b953c87df8e145cb83d447921c